cli/src/cli-rpc-ops.c

```diff
@@ -10975,7 +10976,7 @@ static struct rpc_clnt_procedure gluster_cli_actors[GLUSTER_CLI_MAXVALUE] = {
     [GLUSTER_CLI_DELETE_VOLUME] = {"DELETE_VOLUME", gf_cli_delete_volume},
     [GLUSTER_CLI_START_VOLUME] = {"START_VOLUME", gf_cli_start_volume},
     [GLUSTER_CLI_STOP_VOLUME] = {"STOP_VOLUME", gf_cli_stop_volume},
-    [GLUSTER_CLI_RENAME_VOLUME] = {"RENAME_VOLUME", gf_cli_rename_volume},
+    //[GLUSTER_CLI_RENAME_VOLUME] = {"RENAME_VOLUME", gf_cli_rename_volume},
```

Comment:
You should verify what happens when a defined proc doesn't have a defined handler. Is it correctly ignored ?

Reply:
Yes, I did check and it gave a compilation error if we don't define the handler for it.
So, will removing this also cause compatibility issues, at the time of upgrade? If so, we can simply define the handler to return 0 instead of doing anything.

Reply:
Not defining the entry (or commenting it) shouldn't create a compile error, but the handler will be defined as NULL. We need to make sure that the caller of those handlers correctly handles the case when it's NULL. Another way is to create a dummy function for obsoleted procedures that simply returns an error (an error is better than 0 because we are not doing anything really and no one should be calling it anyway).

For documentation purposes maybe the second approach is better.

Please, also add a comment telling that this procedure is obsoleted and not used anymore (I think the best place is the enum).

Reply:
I think adding a log with `WARNING` level, saying `this feature is depreciated and comment in the enum should do.
Anyways, the way cli works, it will give an error if it finds any command which is not registered.

Reply:
Done.